authors/apps/articles/tests/test_articles.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
+from authors.apps.articles.tests.test_base import ArticleBaseTest
+from rest_framework import status
+
+class ArticleTest(ArticleBaseTest):
+
+    def test_create_article(self):
+        response = self.create_article()
```

Comment:
@Emmanuelaaron I notice you are calling `self.create_article()` in every test_case  in this file; What about calling in this method in your` setUp() `method to avoid this repetition

Reply:
thanks, @alexxsanya for that observation. Let me work on it right away